core/crypto/des.js

```diff
@@ -258,9 +258,74 @@ export default class DES {
         }
         return b;
     }
+}
+
+export class DESECBCipher {
```

Comment:
We should probably stop exporting the old interface so that we use the new one everywhere.